.travis.yml

```
@@ -25,11 +25,12 @@ stages:
   - name: Lint
   - name: Tests
```

You could factorize conditions like this:
```suggestion
  - name: Tests
    if: type = pull_request OR (type = push AND branch =~ /^master|[0-9]+-dev$/)
```
If you like this way, do not apply suggestion by hand. You should edit directly the file.